### DISCUSSION TITLE
how to add all sites to a custom list?

### DISCUSSION THREAD

**Discussion by VioletSkies19** [iptv-org/community#551] [Q&A] [answered]
how do you make a custom channel list with all sites in it? the readme left me confused on how to add sites to the custom channel list xml, specifically where to find these
lang="en" xmltv_id="ArirangTV.kr" site_id="CH_K"

is there a channel list xml already created somewhere i can use?

**Comment by PopeyeTheSai10r** [comment, marked as answer]
Each of the folders have their own channel list. You will need to check each folder or the ones you are interested in https://github.com/iptv-org/epg/tree/master/sites.

See https://github.com/iptv-org/epg/blob/master/CONTRIBUTING.md for more information

**Reply by dacrab** [reply]
so i have all the sites in the "Sites" subfolder but on the "sites.xml" in the startpage is only from one site , is this how it should be?

**Comment by TonyDeGouveia13** [comment]
Tengo una duda, estoy haciendo la lista pero no puedo marcar los canales Espn, solo tiene un punto y no logro marcarlos, así hay otros canales ¿Porque?.
